src/Common/src/System/Diagnostics/Debug.Windows.cs

```diff
@@ -75,6 +77,25 @@ private static void WriteToDebugger(string message)
                     Interop.mincore.OutputDebugString(message ?? string.Empty);
                 }
             }
+
+            private static unsafe bool WriteToStdErr(string message)
+            {
+                IntPtr handle = Interop.mincore.GetStdHandle(Interop.mincore.HandleTypes.STD_ERROR_HANDLE);
```

Comment:
If it doesn't have a stderr handle, it'll return NULL.  If it does but fails for some reason, it'll return INVALID_HANDLE_VALUE.  Hence the subsequent checks for IntPtr.Zero and (IntPtr)(-1).